ladder/teams/README.md

```diff
@@ -0,0 +1,51 @@
+# How Cilium Org Teams Are Managed
+Each Cilium org team that wishes to self-manage its team membership can do so using this directory.
+
+Each team can establish a file in this directory managed by the CodeOwners system with the assistance of 
+the Org admins (or duly deputized members of the org tasked with helping initiating team self-management). 
```

Comment:
It might be helpful to clarify here who the org admins are

Reply:
I'm trying to avoid naming individuals, as this would potentially make this file inaccurate if admins change for some reason.  I guess i can see if I could use a github URL link to the admin list.

Reply:
I have a link to org owners list now. I think that is the best i can do.. until there is a dedicated team for this. And I added some text to make it explicit that you can open a PR to start the process of getting a new file.

Reply:
I think i have this addressed now iwth a link back to other community repo readme.